tests/Museum/MuseumTest.php

```diff
@@ -11,10 +11,20 @@ class MuseumTest extends TestCase
+        $expected = '<div class="card m-4" style="width: 18rem;"><h2 class="card-title text-center mt-3">test</h2><div class="dino-img-container mx-auto" style="background-image: url(https://upload.wikimedia.org/wikipedia/commons/thumb/1/16/Maiasaura_skeleton.jpg/220px-Maiasaura_skeleton.jpg);"></div><div class="card-body d-flex flex-row align-items-center py-5"><div class="food-type d-flex flex-row align-items-center"><img width="50px" alt="Icon to represent Herbivore" src="Icons/herbivore.png"/><p class="card-text">Herbivore</p></div></div></div><div class="card m-4" style="width: 18rem;"><h2 class="card-title text-center mt-3">test</h2><div class="dino-img-container mx-auto" style="background-image: url(https://upload.wikimedia.org/wikipedia/commons/thumb/1/16/Maiasaura_skeleton.jpg/220px-Maiasaura_skeleton.jpg);"></div><div class="card-body d-flex flex-row align-items-center py-5"><div class="food-type d-flex flex-row align-items-center"><img width="50px" alt="Icon to represent Herbivore" src="Icons/herbivore.png"/><p class="card-text">Herbivore</p></div></div></div><div class="card m-4" style="width: 18rem;"><h2 class="card-title text-center mt-3">test</h2><div class="dino-img-container mx-auto" style="background-image: url(https://upload.wikimedia.org/wikipedia/commons/thumb/1/16/Maiasaura_skeleton.jpg/220px-Maiasaura_skeleton.jpg);"></div><div class="card-body d-flex flex-row align-items-center py-5"><div class="food-type d-flex flex-row align-items-center"><img width="50px" alt="Icon to represent Herbivore" src="Icons/herbivore.png"/><p class="card-text">Herbivore</p></div></div></div>';
         $this->assertEquals($expected, $result);
     }
```

Comment:
Could you add additional tests here please? First one would be testing what happens when you give it an array of non Dinosaur objects, to make sure you get that error message, and the second would be giving it an empty array with no Dinosaurs at all, to test that error message as well.